tests/test_decode.py

```diff
@@ -597,7 +618,7 @@ def test_empty_channel(self):
         self.assertEqual(content["msg_type"], 18)
         self.assertEqual(content["repeat"], 0)
         self.assertEqual(content["mmsi"], "1000000000")
-        self.assertEqual(content["speed"], 1023)
+        self.assertEqual(content["speed"], 102.3)
```

Comment:
actually speed value of 1023 means not available, it does seem a bit strange seeing it as an actual float. But consistent with API use which doesn't explicitly deal with not provided values.

Reply:
Realistically 1023 should be returned as `None` if it represents not available.

Reply:
I think that this is something that the application should decide. I would like to return the raw values as often as possible. Also both [tbsalling](http://ais.tbsalling.dk/) and [aggsoft](https://www.aggsoft.com/ais-decoder.htm), two decoders that I used heavily during my development, return `102.3` as a value. I think that it would be wise to be consistent here.